src/imgtools/coretypes/Mask_VectorMask.md

```
@@ -0,0 +1,227 @@
+so we would know things like `object count`, `volume`, `diameter` of our 
+saved masks, without having to open the image and mask in a viewer. 
+
+Similar to how we [calculate stats from the new base `MedImage` class](base_medimage.py#L113-123)
+we can build these into the new mask classes for easy access.
+
+
+## Proposed data structure
+
+```python
+class Mask(MedImage):
+	"""A scalar label mask image with sitk.LabelUInt8 pixel type."""
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)